sources/api/migration/migration-helpers/src/common_migrations.rs

```diff
@@ -703,6 +700,7 @@ impl Migration for ReplaceTemplateMigration {
             self.update_template_and_data(
                 // Clone the input string; we need to give the function mutable access to
                 // the structure that contains the string, so we can't pass a reference into the structure.
+                #[allow(clippy::unnecessary_to_owned)]
                 &data.to_owned(),
```

Comment:
This doesn't work?

```suggestion
                &data,
```

Reply:
That's what clippy suggests, but it leads to this compiler error:
```
error[E0502]: cannot borrow `input` as mutable because it is also borrowed as immutable
   --> api/migration/migration-helpers/src/common_migrations.rs:706:17
    |
690 |         if let Some(input_value) = input.data.get(self.setting) {
    |                                    ---------------------------- immutable borrow occurs here
...
700 |             self.update_template_and_data(
    |                  ------------------------ immutable borrow later used by call
...
706 |                 &mut input,
    |                 ^^^^^^^^^^ mutable borrow occurs here
```